composer.json

```diff
@@ -23,6 +23,7 @@
     "monolog/monolog": "~1.11",
     "nikic/php-parser": "^2 || ^3",
     "silverstripe/config": "^1@dev",
+    "silverstripe/assets": "^1@dev",
```

Comment:
Yeah, there is quite a bit of work to de-couple that at this stage. It may be closer to a 5.0 / 4.late goal. :)